core/src/test/java/org/testcontainers/images/builder/DockerfileBuildTest.java

```diff
@@ -56,12 +64,13 @@ public DockerfileBuildTest(String expectedFileContent, ImageFromDockerfile image
     public void performTest() {
         try (final GenericContainer container = new GenericContainer(image)
             .withStartupCheckStrategy(new OneShotStartupCheckStrategy())
-            .withCommand("cat", "/test.txt")) {
-
+            .withCommand("cat", "/test.txt")) 
+        {
```

Comment:
unrelated change?

Reply:
yes, this is an unrelated change. Just cleaning up formatting to match what you had requested in another review comment since I was changing this file already